packages/api/tests/lambdas/test-sf-scheduler.js

```diff
@@ -139,9 +138,9 @@ test.serial('Sends a message to SQS with the startSF queue if queueName is not d
   t.is(sqsStub.calledOnce, true);
 
   const [targetQueueUrl, targetMessage] = sqsStub.getCall(0).args;
-  t.is(targetQueueUrl, fakeMessageResponse.meta.queues[defaultQueueName]);
-  t.is(targetMessage.cumulus_meta.queueName, defaultQueueName);
-  t.is(targetMessage.meta.queueExecutionLimits[customQueueName], 5);
+  t.is(targetQueueUrl, defaultQueueUrl);
+  t.is(targetMessage.cumulus_meta.queueUrl, defaultQueueUrl);
+  t.is(targetMessage.cumulus_meta.queueExecutionLimits[customQueueUrl], 5);
```

Comment:
I know this was originally here, but it seems kind of bad to test that we are getting the queue execution limits in a seemingly unrelated test for the use of a default URL.

Reply:
i.e. feels bad that there isn't really a dedicated test. Not very self-documenting.

Reply:
^ concur, I had a line with @markdboyd  offline to talk about how this was tested in a deployment scenario, and discuss if we should write one.

Reply:
I agree that this shouldn't be tested here. But I'm not sure where the proper place is to test this.

I could add a test to this file to show that whatever is in the message template shows up in the message:

https://github.com/nasa/cumulus/blob/master/packages/message/tests/test-Build.js#L67

But are you talking about testing that the deployed message template itself has the right contents?

Reply:
I think my original concern was two-fold, the first more along the lines of us not having an explicit test for the custom queue behavior and/or the template content, but startSfSpec probably has enough unless this gets more complex. 

Re: unit tests, I'll let @nemreid  speak to his concern, but 'I could add a test to this file to show that whatever is in the message template shows up in the message' seems like a workable approach for the unit test concern.

Reply:
^ Yes, that.

Reply:
And.... @markdboyd  has pointed out I'm entirely incorrect re: the integration test.   So ignore that first comment re: int tests.

Reply:
Updated unit tests:

1d232833b967d7ce0cb2d14f7dc132c0f1ea0703 
42ed214e5aeeddb27aeabfdd5be61de4fba7f6c0

And we do have an integration test for the queue throttling:

https://github.com/nasa/cumulus/blob/master/example/spec/parallel/startSF/startSfSpec.js#L277